app/containers/App/reducer.js

```diff
@@ -55,6 +57,19 @@ const appReducer = (state = initialState, action) =>
       draft.ready = ready;
     };
 
+    const mapSpecialVaults = (vault) => {
```

Comment:
Manual overrides for special vaults in the reducer

Reply:
for icons, we can use the new path `https://raw.githubusercontent.com/yearn/yearn-assets/master/icons/multichain-tokens/1/${checksumAddress}/logo-128.png` since we are deprecating the one without multichain in this PR https://github.com/yearn/yearn-finance/pull/439

Reply:
Updated